tests/test_destroy.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
         self.snap_device = "/dev/elastio-snap{}".format(self.minor)
 
     def test_destroy_nonexistent_device(self):
-        self.assertEqual(elastio_snap.destroy(self.minor), errno.ENOENT)
+        self.assertEqual(elastio_snap.destroy(self.minor, retries=1, debug=False), errno.ENOENT)
```

Comment:
This is an example that confirms my words above. Retrying only for certain errors that are not `ENOENT` is a good idea)

Reply:
Fixed.